docs/pages/networking/livestreaming.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Livestreaming
+layout: default
+parent: Networking
+---
+
+# Livestream
+
+Stabilizer utilizes a directed UDP port for livestreaming data. The destination of the stream must
+be configured via a settings configuration to enable streaming, and is off by default. Once a remote
+destination for the data stream is specified, Stabilizer will immediately begin streaming live data
+to the provided address.
+
+Currently, Stabilizer only supports streaming of raw ADC/DAC data for both channels.
+
+## Data Format
+
+Data sent via UDP is sent in "blocks". Each block is a single batch of ADC/DAC codes from an
+individual DSP processing routine. Each block is assigned a unique 16-bit identifier. The identifier
+increments by one for each block and rolls over. All blocks in a single packet are guaranteed to
+contain sequential identifiers.
+
+All data is transmitted in network-endian format.
```

Comment:
```suggestion
All data is transmitted in network-endian (big-endian) format.
```

But this will and should change for performance. Better to move these docs into the source code to increase the probablity that they will be updated.